hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/CompactStreamSerializerTest.java

```diff
@@ -500,7 +505,9 @@ public void testFieldOrderFixedSize() throws IOException {
 
         SchemaWriter writer = new SchemaWriter("typeName");
 
-        ReflectiveCompactSerializer reflectiveCompactSerializer = new ReflectiveCompactSerializer();
+        CompactStreamSerializer compactStreamSerializer = mock(CompactStreamSerializer.class);
+        when(compactStreamSerializer.canBeSerializedAsCompact(any())).thenReturn(true);
+        ReflectiveCompactSerializer reflectiveCompactSerializer = new ReflectiveCompactSerializer(compactStreamSerializer);
```

Comment:
Can `CompactTestUtil.getSchemasFor` be used instead of these lines?

Reply:
Yes, that could simplify the tests a bit. I did that in the latest commit